src/rules/enforce-fieldpath-syntax-in-docsetter.ts

```
@@ -0,0 +1,239 @@
+          prop.type === AST_NODE_TYPES.Property &&
+          prop.key.type === AST_NODE_TYPES.Identifier &&
+          prop.key.name === 'id',
+      );
+
+      const flattenedProperties = flattenObject(node, sourceCode);
+
+      // Start with the id property if it exists
+      let result = '{\n';
+      if (idProperty && idProperty.type === AST_NODE_TYPES.Property) {
+        result += `  id: ${sourceCode.getText(idProperty.value)},\n`;
+        delete flattenedProperties['id'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)